addon/templates/components/frost-link.hbs

```diff
@@ -1,26 +1,26 @@
 {{#if (eq design 'info-bar')}}
-  <div class='content '>
-    <div class=' text'>
-      <div class='svg'>{{frost-icon icon=icon}}</div>
-      {{yield}}
-      </div>
-  </div>
+    <div class='content '>
+        <div class=' text'>
+            <div class='svg'>{{frost-icon icon=icon}}</div>
+          {{#if linkTitle}}{{linkTitle}}{{else}}{{yield}}{{/if}}
```

Comment:
This is coming from built in Ember `link-to` component.
https://github.com/emberjs/ember.js/blob/aeeddf7c00b42a340494465252481b5414d87009/packages/ember-glimmer/lib/components/link-to.js#L795